doc/hardware/porting/board_porting.rst

```diff
@@ -361,21 +519,28 @@ named ``plank``:
 
      endif # BOARD_PLANK
 
```

Comment:
`  Not all boards have a :file:`Kconfig.defconfig` file.`

Reply:
added above code block, so it follows same structure as the `Kconfig` file description.